.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "src/clp"]
 	path = src/clp
-	url = https://github.com/y-scope/clp.git
+	url = https://github.com/junhaoliao/clp.git
```

Comment:
These shall not be merged

Reply:
The PR is merged with the latest CLP code: #55